base_user_api_key_security/views/res_users_views.xml

```diff
@@ -0,0 +1,12 @@
+<odoo>
+    <record id="view_users_form_simple_modif_extend" model="ir.ui.view">
+        <field name="name">res.users.preferences.form.api_key_restrict</field>
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.view_users_form_simple_modif" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='api_key']" position="attributes">
```

Comment:
We can remove the dependency on the PR to the core Odoo for now (to expedite our process).
```suggestion
            <xpath expr="//field[@name='api_key_ids']/../.." position="attributes">
```